### PR TITLE
#83 - clean up URLs and remove some unnecessary AssaultCube reference…

### PR DESCRIPTION
**Pull request by ScalaWilliam**
…s. Add RELEASENOTES.md and UNINSTALL.md

config/opt seems redundant too.

### CODE REVIEW

File: source/vcpp/buildEnv/af.nsi [hunk 1]
```diff
@@ -546,21 +546,21 @@ Section "ActionFPS ${AC_FULLVERSION}" AC
     WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${AC_SHORTNAME}" "UninstallString" '"$INSTDIR\Uninstall.exe"'
     WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${AC_SHORTNAME}" "DisplayIcon" '"$INSTDIR\icon.ico"'
     WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${AC_SHORTNAME}" "HelpLink" "$INSTDIR\README.html"
-    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${AC_SHORTNAME}" "URLInfoAbout" "http://assault.cubers.net"
-    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${AC_SHORTNAME}" "URLUpdateInfo" "http://assault.cubers.net/download.html"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${AC_SHORTNAME}" "URLInfoAbout" "https://actionfps.com"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${AC_SHORTNAME}" "URLUpdateInfo" "https://actionfpscom"
```
Comment (lucasgautheron) [inline source/vcpp/buildEnv/af.nsi]: .com, missing dot

File: UNINSTALL.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+If you have had any issues, please report them to
+<a href="https://github.com/lucasgautheron/ActionFPS-Game/issues">ActionFPS-Game GitHub Issues</a>.
```
Comment (lucasgautheron) [inline UNINSTALL.md]: ```[ActionFPS-Game GitHub Issues](https://github.com/lucasgautheron/ActionFPS-Game/issues)```?

Comment (ScalaWilliam) [inline UNINSTALL.md]: can't stand the syntax ;)

**Comment by ScalaWilliam** [comment]
I don't know how to kill appveyor bot

**Pull request merged**